INFO  | File run_exp_CCN_Project.py | Line 15    | Writing log file to ./exp/GTS_test_1217_173731/log_exp_1010.txt
INFO  | File run_exp_CCN_Project.py | Line 16    | Exp instance id = GTS_test_1217_173731
ERROR | File run_exp_CCN_Project.py | Line 25    | Traceback (most recent call last):
  File "/Users/juhyeonkim/Dropbox/내 Mac (JuHyeonui-iMac.local)/Documents/GraphStructureLearning/run_exp_CCN_Project.py", line 20, in main
    runner.train()
  File "/Users/juhyeonkim/Dropbox/내 Mac (JuHyeonui-iMac.local)/Documents/GraphStructureLearning/runner/CCN_Project_runner.py", line 151, in train
    _, _, loss = self.model(inputs=data_batch.x, targets=target,
  File "/opt/anaconda3/envs/GraphStructureLearning/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/Users/juhyeonkim/Dropbox/내 Mac (JuHyeonui-iMac.local)/Documents/GraphStructureLearning/models/GTS/GTS_model_for_CNN_Project.py", line 58, in forward
    loss = self.loss_func(outputs, targets)
  File "/opt/anaconda3/envs/GraphStructureLearning/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/opt/anaconda3/envs/GraphStructureLearning/lib/python3.9/site-packages/torch/nn/modules/loss.py", line 1150, in forward
    return F.cross_entropy(input, target, weight=self.weight,
  File "/opt/anaconda3/envs/GraphStructureLearning/lib/python3.9/site-packages/torch/nn/functional.py", line 2846, in cross_entropy
    return torch._C._nn.cross_entropy_loss(input, target, weight, _Reduction.get_enum(reduction), ignore_index, label_smoothing)
RuntimeError: Expected floating point type for target with class probabilities, got Long

